classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php

```diff
@@ -42,11 +42,11 @@ function initData() {
 		// Retrieve most recent reviewer comments, one private, one public.
 		$submissionCommentDao = DAORegistry::getDAO('SubmissionCommentDAO');
 
-		$submissionComments = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getReviewerId(), $reviewAssignment->getSubmissionId(), $reviewAssignment->getId(), true);
+		$submissionComments = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getSubmissionId(), $reviewAssignment->getReviewerId(), $reviewAssignment->getId(), true);
 		$submissionComment = $submissionComments->next();
 		$this->setData('comment', $submissionComment?$submissionComment->getComments():'');
```

Comment:
Nope, the Reviewer's interface still just shows the most recent comment. This is kind of a legacy data concern -- in OJS2 multiple comments were allowed, and in OJS3 only one is.